api/schemas.go

```diff
@@ -29,6 +29,7 @@ type integrationSchema int
 const (
 	None integrationSchema = iota
 	AlertChannels
+	AlertRules
```

Comment:
can we inject constants like this? Im concerned we are going to change the number from the `iota` and something could get messed up 🤔

Reply:
We don't make use the numerical value, we're really just trying to create an enumerated type for supported services.